cypress/support/config/envs.js

```diff
@@ -10,6 +10,16 @@ const config = {
     standaloneErrorPages: false,
     alwaysCheckForFallback: true,
   },
+  stage: {
+    baseUrl: 'https://www.stage.bbc.com',
+    dataUrl: 'https://www.stage.bbc.com',
+    assetUrl: 'https://news.stage.files.bbci.co.uk/include/articles/public',
```

Comment:
Unless someone is currently setting this up, `news.stage.files` does not exist I don't think.